src/js/components/DataTable/StyledDataTable.js

```diff
@@ -82,7 +81,9 @@ const StyledDataTableRow = styled(TableRow)`
     props.onClick &&
     `
     cursor: pointer;
-  `}
+  `}  
+
+  // Could remove: changes nothing UI-wise
```

Comment:
Should we remove this styling block and comment?

Reply:
I haven't ran through all of the other DataTable stories with this removed in a while so I will look into this today to see if it really doesn't affect anything else